.github/workflows/auto-upgrade-certified-connectors-cdk.yml

```diff
@@ -0,0 +1,149 @@
+name: Auto Upgrade CDK for Certified Connectors
+on:
+  schedule:
+    # Run at 16:37 UTC on the first of every month
+    - cron: "37 16 1 * *"
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  list-certified-connectors:
+    name: List Certified Connectors
+    runs-on: ubuntu-24.04
+    outputs:
+      connectors: ${{ steps.list-connectors.outputs.connectors }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4
+
+      - name: Install yq
+        run: sudo snap install yq
+
+      - name: List certified connectors
+        id: list-connectors
+        run: |
+          json_array=$(tools/bin/bulk-cdk-auto-upgrade/list-connectors-to-upgrade.sh)
+          echo "connectors=$json_array" >> $GITHUB_OUTPUT
+          echo "Found bulk CDK connectors to upgrade: $json_array"
+
+  upgrade-connector-cdk:
+    name: Upgrade CDK for ${{ matrix.connector }}
+    needs: list-certified-connectors
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        connector: ${{ fromJson(needs.list-certified-connectors.outputs.connectors) }}
+      fail-fast: false
+      max-parallel: 5
+    steps:
+      - name: Authenticate as GitHub App
+        uses: actions/create-github-app-token@67018539274d69449ef7c02e8e71183d1719ab42 # v2
+        id: app-token
+        with:
+          owner: "airbytehq"
+          repositories: "airbyte"
```

Comment:
if we are planning to call this from the enterprise repo we probably need to be able to set this with workflow call input

Reply:
maybe too late, but probably if we don't set this field it would default to the current repo, which should work for both the oss and the enterprise repo

Reply:
missed this earlier, but I realized we need the repo in the changelog step anyway (b/c enterprise connectors don't have a changelog >.>)

so I guess I'll keep it as-is